File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/internal/data/framework/api/apifile/IFileApi.java
package ru.usedesk.chat_sdk.internal.data.framework.api.apifile;

import androidx.annotation.NonNull;

import java.util.List;

import okhttp3.MultipartBody;
import ru.usedesk.chat_sdk.internal.data.framework.api.apifile.entity.FileResponse;

public interface IFileApi {
    @NonNull
    FileResponse post(@NonNull String baseUrl,
                      @NonNull List<MultipartBody.Part> request) throws Exception;
}

File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/data/repository/api/loader/apifile/IFileApi.kt
package ru.usedesk.chat_sdk.data.repository.api.loader.apifile

import okhttp3.MultipartBody
import ru.usedesk.chat_sdk.data.repository.api.loader.apifile.entity.FileResponse

internal interface IFileApi {
    fun post(baseUrl: String, request: List<MultipartBody.Part>): FileResponse
}